02_google_01_cloud.md

```diff
@@ -0,0 +1,33 @@
+
+# What is the cloud
```

Comment:
Forgot that the google set up stuff is kinda necessary for some of the other items so I added that in here after chromebook but before R.